### PR TITLE
Fix test reporter workflow for PRs

### LINKED ISSUE #102: Tests broken, again
https://github.com/qurator-spk/dinglehopper/actions/runs/7285741554/job/19868158221?pr=101

May be because it was a PR.

### PR DESCRIPTION
**Pull request by sadra-barikbin**
Fixes #102

### CODE REVIEW

File: src/dinglehopper/align.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def align(t1, t2):
     return seq_align(s1, s2)
 
 
-def score_hint(er: float, n: int) -> Optional[int]:
```
Comment (sadra-barikbin) [inline src/dinglehopper/align.py:18]: Using `Optional` implies that the function might not return anything in some circumstances but here we might return `None`.

File: src/dinglehopper/character_error_rate.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import unicodedata
-from typing import Tuple, List
+from typing import List, Tuple
```
Comment (sadra-barikbin) [inline src/dinglehopper/character_error_rate.py:2]: To have them alphabetically sorted.

File: src/dinglehopper/align.py (hunk 1 above)
Comment (kba) [reply src/dinglehopper/align.py:18]: `Optional[int]` and `Union[int, None]` are equivalent. With Python [3.10+](https://peps.python.org/pep-0604/) you could also write `int|None`

**Comment by mikegerber** [comment]
It's merged in 788868b, GH does not seem to pick it up here after my rebase.

**Pull request closed without merging**